### PR TITLE
[Samples] Remove duplicate scenarios and update properties

### LINKED ISSUE #7516: {Freeway-Bug} [Voice Access> ActivityUpdate.json> Set Due Date]: Name is not defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro steps:**
1. Turn on Voice access.
2. Open Adaptive Cards UWP Application.
3. Command: Show numbers
4. Select particular number for 'ActivityUpdateWithLabel.json'
5. Command: Move to 'Set due date'
6. Command: Click
7. Now, try to access the edit field of calendar date picker
8. Observe the issue.

**Actual Result:**
Name is not defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.

**Expected Result:**
Proper name should be defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.

**User Impact:**
Voice access and Screen reader user will have difficulty in interacting with the controls as they will not be able to know the purpose of the control.

**MAS reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

MAS4 1 2 Name property of the calendar date picker under activityupdate is null
MAS4 2 1 Name property is not defined for calendar date picker

### Card JSON

```JSON
NA
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### LINKED ISSUE #7518: [CalendarReminder.json>Snooze for]: Voice access user is not able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**
1. Turn on Voice access.
2. Open Adaptive Cards UWP application.
3. Command: Show Numbers
4. Select particular number for 'CalendarReminder.json' 
5. Command: Move to Snooze for
6. Observe the issue.

**Actual Result:**
Voice access user is not able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.

**Expected Result:**
Voice access user should be able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.

**Observation:**
'Click Snooze for' command is also not working.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will get impacted as they will not be able to interact with the control easily and accessing through numbers would take more time for them.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

MAS4 3 1 'Move to Snooze for ' command is not getting performed

https://user-images.githubusercontent.com/104483005/173425823-76f4029e-5172-4974-a9b1-db818cc1448a.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### PR DESCRIPTION
**Pull request by anna-dingler**
# Related Issue

Progress towards #7992 - still need to update `Elements` and `Tests`
Fixes #7518
Fixes #7516

# Description

Many of our scenario cards have version discrepancies (ex: v1.0 card contains v1.3 properties).

I've updated the cards to use the proper version and removed cards that had multiple versions.

## Notable Changes
- Added `CaptionSource` to the ProductVideo sample

# How Verified

Verified cards in the Designer.
Requires testing on the Samples and Designer pages of the website.

image

image

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/8010)

### CODE REVIEW

File: samples/Templates/Scenarios/ProductVideo.template.json [hunk 1]
```diff
@@ -1,16 +1,28 @@
 {
-        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-        "type": "AdaptiveCard",
-        "version": "1.1",
-        "fallbackText": "This card requires Media to be viewed. Ask your platform to update to Adaptive Cards v1.1 for this and more!",
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.6",
+    "fallbackText": "This card requires Media to be viewed. Ask your platform to update to Adaptive Cards v1.1 for this and more!",
```
Comment (paulcam206) [inline samples/Templates/Scenarios/ProductVideo.template.json:5]: version needs to be updated

File: samples/v1.5/Scenarios/InputsWithValidation.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
-	"version": "1.0",
+	"version": "1.5",
```
Comment (paulcam206) [inline samples/v1.5/Scenarios/InputsWithValidation.json:4]: `1.3`?

**Review by paulcam206 — CHANGES_REQUESTED** [review]
some minor issues

**Comment by paulcam206** [comment]
would love to see a screenshot of the Samples page on the site with these changes :)

**Comment by paulcam206** [comment]
ooh! while you're in here, can you move `v1.2\Scenarios\SimpleFallback.json` to `v1.2\Tests`?

File: samples/v1.5/Scenarios/InputsWithValidation.json (hunk 2 above)
Comment (anna-dingler) [reply samples/v1.5/Scenarios/InputsWithValidation.json:4]: We use `tooltip` within this one on `Action.OpenUrl`

**Comment by anna-dingler** [comment]
@paulcam206 Do we want to remove the `FlightUpdate` sample?

Right now, we have `FlightUpdate` in v1.0 and `FlightUpdateTable` in v1.5.

File: samples/Templates/Scenarios/CalendarReminder.template.json [hunk 3]
```diff
@@ -1,38 +1,31 @@
...
-	"version": "1.0",
+	"version": "1.3",
 	"speak": "<s>Your  meeting about \"Adaptive Card design session\"<break strength='weak'/> is starting at ${formatDateTime(start.dateTime, 'HH:mm')}pm</s><s>Do you want to snooze <break strength='weak'/> or do you want to send a late notification to the attendees?</s>",
 	"body": [
 		{
 			"type": "TextBlock",
 			"text": "${summary}",
 			"size": "Large",
 			"weight": "Bolder",
-			"wrap": true,
```
Comment (almedina-ms) [inline samples/Templates/Scenarios/CalendarReminder.template.json:12]: for accessibility reasons having wrap in this element may be required

File: samples/Templates/Scenarios/ExpenseReport.template.json [hunk 4]
```diff
@@ -209,30 +204,31 @@
...
-							"verticalContentAlignment": "Center",
+							"verticalContentAlignment": "center",
 							"items": [
 								{
 									"type": "Image",
 									"id": "chevronDown${$index}",
 									"url": "https://adaptivecards.io/content/down.png",
 									"width": "20px",
-									"altText": "${description} $${total} collapsed"
+									"altText": "Details collapsed"
```
Comment (almedina-ms) [inline samples/Templates/Scenarios/ExpenseReport.template.json:224]: is this change intentional?

File: samples/Templates/Scenarios/CalendarReminder.template.json (hunk 3 above)
Comment (anna-dingler) [reply samples/Templates/Scenarios/CalendarReminder.template.json:12]: Sounds good, I'll make sure wrap is included on the TextBlocks

File: samples/Templates/Scenarios/ExpenseReport.template.json (hunk 4 above)
Comment (anna-dingler) [reply samples/Templates/Scenarios/ExpenseReport.template.json:224]: This was probably a difference between `ExpenseReport` and `ExpenseReportWithLabels`. Most of the changes in this file occurred because I **deleted** `ExpenseReport` and **renamed** `ExpenseReportWithLabels`

File: samples/Templates/Scenarios/FoodOrder.template.json [hunk 5]
```diff
@@ -1,174 +1,155 @@
...
-        }
-    ]
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.3",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Your registration is almost complete",
+			"size": "Medium",
```
Comment (almedina-ms) [inline samples/Templates/Scenarios/FoodOrder.template.json:9]: other files have changed words like Medium and Bolder for their lowercase versions, for consistency reasons we should keep them the same across files

File: samples/Templates/Scenarios/ImageGallery.template.json [hunk 6]
```diff
@@ -1,61 +1,61 @@
...
-        }
-    ]
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Here are some cool photos",
+			"size": "Large",
```
Comment (almedina-ms) [inline samples/Templates/Scenarios/ImageGallery.template.json:9]: same here

File: samples/Templates/Scenarios/FoodOrder.template.json (hunk 5 above)
Comment (anna-dingler) [reply samples/Templates/Scenarios/FoodOrder.template.json:9]: @almedina-ms Good call out on consistency. Do we want to keep the uppercase version or lowercase version?

File: samples/v1.5/Scenarios/RestaurantOrder.json [hunk 7]
```diff
@@ -1,105 +1,105 @@
...
\ No newline at end of file
+{
+	"type": "AdaptiveCard",
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"version": "1.5",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Malt & Vine Order Form",
+			"size": "Large",
```
Comment (almedina-ms) [inline samples/v1.5/Scenarios/RestaurantOrder.json:9]: caps in size and weight here too

File: samples/v1.5/Scenarios/FlightUpdateTable.json [hunk 8]
```diff
@@ -1,263 +1,263 @@
...
+			"rows": [
+				{
+					"type": "TableRow",
+					"cells": [
+						{
+							"type": "TableCell",
+							"items": [
+								{
+									"type": "Image",
+									"size": "Small",
```
Comment (almedina-ms) [inline samples/v1.5/Scenarios/FlightUpdateTable.json:28]: same here

File: samples/v1.5/Scenarios/ApplicationLogin.json [hunk 9]
```diff
@@ -1,47 +1,41 @@
...
-    ]
-}
\ No newline at end of file
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.5",
+	"body": [
+		{
+			"type": "TextBlock",
+			"size": "Medium",
```
Comment (almedina-ms) [inline samples/v1.5/Scenarios/ApplicationLogin.json:8]: same here

File: samples/Templates/Scenarios/FoodOrder.template.json (hunk 5 above)
Comment (anna-dingler) [reply samples/Templates/Scenarios/FoodOrder.template.json:9]: @almedina-ms 
I created another PR with the enum lowercase changes - https://github.com/microsoft/AdaptiveCards/pull/8029
I'll go ahead and resolve the other comments, but keep this one open for tracking :)

**Comment by paulcam206** [comment]
> @paulcam206 Do we want to remove the `FlightUpdate` sample?
> 
> Right now, we have `FlightUpdate` in v1.0 and `FlightUpdateTable` in v1.5.

If the table version is as accessible or more accessible than FlightUpdate, then I think the old one should be pulled :)

**Review by paulcam206 — APPROVED** [review]

**Comment by anna-dingler** [comment]
> > @paulcam206 Do we want to remove the `FlightUpdate` sample?
> > Right now, we have `FlightUpdate` in v1.0 and `FlightUpdateTable` in v1.5.
> 
> If the table version is as accessible or more accessible than FlightUpdate, then I think the old one should be pulled :)

Seems like Narrator works better with the old one, so I'll leave it
